index.html

```diff
@@ -284,17 +327,49 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		        
 		    </section><section class="bg-secondary">
 		        <div class="container">
-		            <div class="row mb64 mb-xs-24">
-		                <div class="col-sm-12 text-center spread-children-large">
```

Comment:
I see you have deleted the images and code added by other students. That is not acceptable. If you are making a change in style, you can't just add your information and delete other's

Reply:
Oh very sorry i didn't knew it. When i already submitted and it says that there is a Conflict.